elastic/indexer/elastic-indexer.go

```diff
@@ -121,6 +127,23 @@ func (i *Indexer) runStep() error {
 	return nil
 }
 
+func (i *Indexer) checkAllResultMessagesAreValid(result *bulkResult) error {
+	for _, bulkResultOperation := range result.Items {
+		if bulkResultOperation.Index.ID != "" {
+			return checkHttpStatus(bulkResultOperation.Index.Status)
+		} else if bulkResultOperation.Update.ID != "" {
+			return checkHttpStatus(bulkResultOperation.Update.Status)
```

Comment:
This way`checkAllResultMessagesAreValid` will return `error`/`nil` from the first item, you should only return if the `error` returned by  `checkHttpStatus ` is not nil